simple_utils.py

```
@@ -0,0 +1,12 @@
+# simple_utils.py - A tiny utility library
+
+def reverse_string(text):
+    """Reverses the characters in a string."""
+    return text[::-1]
+
+def count_words(sentence):
```

Ruff (0.14.3)

Unused function argument: `sentence`

(ARG001)